3-0-spring-mvc/src/main/java/com/bobocode/mvc/service/NoteService.java

```diff
@@ -0,0 +1,22 @@
+package com.bobocode.mvc.service;
+
+import com.bobocode.mvc.model.Note;
+import com.bobocode.mvc.storage.NoteStorage;
+import lombok.Data;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Data
```

Comment:
It's not a data, it's a stateless service. Please use `@RequiredArgsConstructor` only

Reply:
@erneman1 please remove the service completely for this exercise